src/transaction.rs

```diff
@@ -0,0 +1,23 @@
+use core::convert::TryFrom;
+
+use anyhow::Error;
```

Comment:
use crate's error type here is more advisable. this is not just a library project but substrate WASM runtime part.

Reply:
Yeah, if we leave the file inside the pallet, that change will probably be needed.
I've added a comment about it, as Karlo may decide to move this file into the backend.

Reply:
So, todo left, I'd keep it right now 'as is'.